README.md

```diff
@@ -62,25 +56,3 @@ This project is built using [Vue 3](https://v3.vuejs.org/guide/introduction.html
 - [file-saver](https://www.npmjs.com/package/file-saver) to create and download files locally
 - [moment](https://www.npmjs.com/package/moment) for formatting date and time
 - [bytes](https://www.npmjs.com/package/bytes) for formatting bytes to human readable values
```

Comment:
Usage, Support, Contributing Guide, and License sections are getting removed in this PR.

Reply:
Creating a fresh PR.  Is there a way to delete abandoned PRs?